python_modules/libraries/dagster-dbt/dagster_dbt_tests/dbt_projects/test_dagster_asset_checks/models/schema.yml

```
@@ -86,3 +86,20 @@ models:
         description: Amount of the order (AUD) paid for by gift card
         tests:
           - not_null
+
+  - name: fail_tests_model
+    columns:
+      - name: id
+        tests:
+          - unique:
+              config:
+                error_if: ">10"
```

Just to be explicit that we want to skip this error condition?


```suggestion
                severity: error # This should never error, and instead we warn at run time
                error_if: "=-1" 
```